corehq/apps/cloudcare/templates/formplayer/case_list.html

```diff
@@ -3,6 +3,11 @@
 <script type="text/template" id="case-view-list-template">
     <h2 id="title"><%= title %></h2>
     <table class="table table-hover table-striped table-bordered">
+
+        <% if (tiles) { %>
+        <div class="wrapper"></div>
```

Comment:
So if you're not using tiles, you have table cells inside of table rows inside of a table, but if you are using tiles, you end up with divs inside of table rows inside a div inside a table? That's pretty weird semantically, though I don't know that it'll cause you any concrete problems.